src/main/java/dropwizard/springboot/dropwizard/to/springboot/entity/UserEntity.java

```diff
@@ -0,0 +1,61 @@
+package dropwizard.springboot.dropwizard.to.springboot.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "tbl_user")
+public class UserEntity {
+    @Id
+    @Column(updatable = false, nullable = false, insertable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)

```

Comment:
Prefer `sequence` instead of `identity`. Relatively complex, but will give you a better understanding.